src/SourceLocator/ComposerSourceLocator.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace BetterReflection\SourceLocator;
+
+use Composer\Autoload\ClassLoader;
```

Comment:
Maybe it's just an extreme case tho, as it assumes the composer autoloader (fair assumption)